src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -623,33 +625,33 @@ event::ptr primitive_inst::execute(const std::vector<event::ptr>& events) {
             return ev;
         }
 
-        if (!is_valid_fusion()) {
-            auto subgraph = get_unfused_subgraph();
-
-            for (auto& d : _deps) {
-                if (!d.first->get_node().is_type<data>()) {
-                    auto allocated_mem = d.first->output_memory_ptr();
-                    auto actual_input_layout = d.first->get_output_layout();
-                    auto& engine = _network.get_engine();
-                    // Need to use actual layout, not the fake aligned memory layout
-                    auto actual_mem = engine.reinterpret_buffer(*allocated_mem, actual_input_layout);
-                    subgraph->set_input_data(d.first->id(), actual_mem);
+            if (!is_valid_fusion()) {
```

Comment:
Looks like unexpected indentation change

Reply:
Yes, you're right. I remove this changes now. thanks.